connect-examples/v2/php_checkout/main.css

```diff
@@ -3,6 +3,14 @@
     margin: 0 auto;
 }
 
+#confirmation {
+  width: 500px;
```

Comment:
I think changing the default font of this page would make a lot of difference. How about adding something like: `font-family: "HelveticaNeue-Bold", Helvetica, Arial, sans-serif;`